packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -13,18 +13,28 @@ Event hooks are outbound calls from Okta, sent when specified events occur in yo
 
-You can have a maximum of 10 active and verified event hooks in your org at any point. Each event hook can be configured to deliver multiple event types.
+Before the introduction of event hooks, polling the [System Log API](/docs/api/resources/system_log/) was the only method your external software systems could use to detect the occurrence of specific events in your Okta org; event hooks provide an Okta-initiated push notification.
+
+You can have a maximum of 10 active and verified event hooks set up in your org at any time, and each event hook can be configured to deliver multiple event types.
 
 ## Which Events are Eligible?
 
-During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures. To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
+During the initial configuration procedure for an event hook, you specify which event types you want the event hook to deliver. The event types that can be specified are a subset of the event types that the Okta System Log captures.
+
+To see the list of event types currently eligible for use in event hooks, query the Event Types catalog with the query parameter `event-hook-eligible`:
 
 [https://developer.okta.com/docs/api/resources/event-types/?q=event-hook-eligible](/docs/api/resources/event-types/?q=event-hook-eligible)
 
+For general information on how Okta encapsulates events, see the [System Log API](/docs/api/resources/system_log/) documentation.
+
+Examples of available types of events include user lifecycle changes, the completion by a user of a specific stage in an Okta process flows, and changes in Okta objects. You could configure an event hook, for example, to deliver notification of user deactivation events. You could use this to trigger processes you need to execute internally every time a user is deactivated, like updating a record in an HR system, creating a ticket in a support system, or generating an email message.  
+
 ## Requests Sent by Okta
 
+When events occur in your org that match an event type you configured your event hook to deliver, the event hook is automatically triggered and sends a request to your external service. The JSON payload of the request provides information on the event. A sample JSON payload is provided in [Sample Event Delivery Payload](#sample-event-delivery-payload) below.
```

Comment:
Adding "that" for translation

When events occur in your org that match an event type THAT you configured your event hook to deliver, the event hook is automatically triggered and sends a request to your external service. The JSON payload of the request provides information on the event. A sample JSON payload is provided in [Sample Event Delivery Payload](#sample-event-delivery-payload) below.

Reply:
Have updated to use "that".